acasclient/experiment.py

```diff
@@ -18,6 +18,7 @@
 ORIGINAL_EXPT_CODE_KEY = "Original Experiment Code"
 ORIGINAL_SERVER_KEY = "Original Server"
 SOURCE_FILE_KEY = "source file"
+REPORT_FILE_KEY = "annotation file"
```

Comment:
Added a few things to the simplified experiment class to handle getting report file information to support the report file tests